src/users/controller/user.controller.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const userService = require('../services/user.service').services;
+
+function get(req, res) {
+  res.send(userService.getUsers());
+}
```

Comment:
for now it is okay, because users is always exist, but in a real project you should handle error here, because you will take users from db etc.